app/models/manageiq/providers/openshift/inventory/collector/target_collection.rb

```diff
@@ -0,0 +1,15 @@
+class ManageIQ::Providers::Openshift::Inventory::Collector::TargetCollection < ManageIQ::Providers::Openshift::Inventory::Collector
+  include ManageIQ::Providers::Kubernetes::ContainerManager::TargetCollectionMixin
+
+  def inventory(entities)
+    full_inventory = empty_inventory(entities)
+
+    # Fill pods from Targets
+    full_inventory['pod'] = pods
```

Comment:
should pods be here given it's an kubernetes entity?

Reply:
ah, so I am overwriting the whole method for now, we can later separate with k8s vs. OSE